tests/phpunit/tests/AutoUpdate/TestTranslationStatus.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace WPML\PB\AutoUpdate;
+
+use Exception;
+use OTGS\PHPUnit\Tools\TestCase;
+
+/**
+ * @group auto-update
+ */
+class TestTranslationStatus extends TestCase {
+
+	/**
+	 * @test
+	 */
+	public function itGetsAndReturnNullOnException() {
+		\WP_Mock::userFunction( 'WPML\Container\make' )->andThrow( new Exception() );
```

Comment:
@strategio I don't think `make` throws an exception in reality. It just returns null if the class doesn't exist.

Reply:
Right @brucepearson, `make` will return an Exception if it can't resolve the dependencies. I adjusted my code accordingly.